net/znc/Makefile

```diff
@@ -23,6 +23,7 @@ PKG_LICENSE_FILES:=LICENSE
 PKG_USE_MIPS16:=0
 PKG_BUILD_PARALLEL:=1
 
+PKG_CONFIG_DEPENDS := CONFIG_ZNC_ICU
 
```

Comment:
This one doesn't work, as PKG_CONFIG_DEPENDS is set again (with :=) after all the module definitions, overwriting this one. Easiest is to change the bottom one to += so it won't discard the previous value.